packages/web/localizations/en.json

```diff
@@ -209,6 +209,8 @@
     "ibcTransfer": {
       "titleWithdraw": "Withdraw {coinDenom}",
       "titleDeposit": "Deposit {coinDenom}",
+      "congested": "congested",
+      "blocked": "blocked",
```

Comment:
I don't know that this will work for other languages besides english. Sentences may not be composed the same.

I think we need to use whole phrases and replace the entire text in the button. We could prob go with "Channel blocked" and "Channel congested".

Reply:
changed